src/OpenTelemetry.Instrumentation.Owin/README.md

```diff
@@ -59,6 +59,27 @@ OpenTelemetry instrumentation which listens to the OWIN diagnostic events.
         .Build();
 ```
 
+Following list of attributes are added by default on activity. See
+[http-spans](https://github.com/open-telemetry/semantic-conventions/tree/v1.23.0/docs/http/http-spans.md)
+for more details about each individual attribute:
+
+* `error.type`
+* `http.request.method`
+* `http.request.method_original`
+* `http.response.status_code`
+* `http.route`
+* `network.protocol.version`
+* `user_agent.original`
+* `server.address`
+* `server.port`
+* `url.path`
+* `url.query` - By default, the values in the query component are replaced with
+  the text `Redacted`. For example, `?key1=value1&key2=value2` becomes
+  `?key1=Redacted&key2=Redacted`. You can disable this redaction by setting the
+  environment variable
+  `OTEL_DOTNET_EXPERIMENTAL_OWIN_DISABLE_URL_QUERY_REDACTION` to `true`.
+* `url.scheme`
```

Comment:
I think that it needs to be double checked.
For what I see, OWIN is using kind of older semantic convention. You can either document it correctly and list all attributes, or update instrumentation to the new standards (possibly v1.27.0) and the provide this documentation.

eg. `url.query` is not set. In fact `http.url`. Same for `server.address` and `http.host`.